Dockerfile

```diff
@@ -1,7 +1,15 @@
-FROM python:alpine3.6
+FROM alpine:3.7
+COPY build.sh /tmp
+WORKDIR /tmp
+RUN mkdir -p /tmp/nexus_configurator/groovy && \
+    apk update && \
+    apk add curl && \
+    ./build.sh
 
+FROM python:3.6-slim
 COPY . /app
 WORKDIR /app
-RUN pip install pipenv && pipenv install
-ENV SHELL=/bin/sh
-ENTRYPOINT ["pipenv", "run", "nexus_configurator"]
+COPY --from=0 /tmp/nexus_configurator/groovy /app/nexus_configurator/groovy
+RUN pip install .
```

Comment:
this command doesn't obey the [Pipfile.lock](https://github.com/ocadotechnology/nexus-configurator/blob/master/Pipfile.lock) version of dependencies